lib/validate.ts

```diff
@@ -136,7 +136,7 @@ const prettyPrintInfo = <T>(
 
 export const validateSpec = async (specPath: string, options: Options | undefined) =>
   validate(options, async (o) => {
-    const validator = new SemanticValidator(specPath, null, o);
+    const validator = new SemanticValidator(specPath, null);
```

Comment:
@jianyexi , do unified pipeline and static validation pass in the option?

Reply:
It just uses the 'validateSpec'